test/test_torch.py

```diff
@@ -1653,14 +1655,6 @@ def reference(x, k, o3, o32):
         self._test_conv_corr_eq(lambda x, k: torch.xcorr3(x, k), reference)
 
     @unittest.skip("Not implemented yet")
-    def test_xcorr3_xcorr2_eq(self):
```

Comment:
Found it. "full" vs "valid" - https://github.com/torch/torch7/blob/cd8de6c67d/lib/TH/generic/THTensorConv.c#L499-L511